README.md

```diff
@@ -7,13 +7,120 @@ api-filter
 
 Parser/builder for filters from api query parameters.
 
+It is just a parser/builder for filters, it is not a place for business logic so it should be wrapped by your class, if you want to be more strict about filters.
+Same if you want different settings per entity/table, it should be done by specific wrapper around this library.
+
+
 ## Installation
 ```bash
 composer require lmc/api-filter
 ```
 
 
+## Filters
+
+### EQ (=)
+```http request
+GET http://host/endpoint/?field[eq]=value
+GET http://host/endpoint/?field=value
+```
+_All examples ☝️ are equal_
+
+### GT (>)
+```http request
+GET http://host/endpoint/?field[gt]=value
+```
+
+### GTE (>=)
+```http request
+GET http://host/endpoint/?field[gte]=value
+```
+
+### LT (<)
+```http request
+GET http://host/endpoint/?field[lt]=value
+```
+
+### LTE (<=)
+```http request
+GET http://host/endpoint/?field[lt]=value
+```
+
+### IN
+```http request
+GET http://host/endpoint/?type[in][]=one&type[in][]=two
+```
+_☝️ is not implemented yet_
+
+
 ## Usage
+For example lets have query parameters from following request
+```http request
+GET http://host/endpoint/?field=value
+```
+
+### Initialization
+```php
+// in DI container/factory
+$apiFilter = new ApiFilter();
+$apiFilter->registerApplicator(...);  // optional
```

Comment:
If this is optional, I don't understand when should I use this.

The same for the "addEscape".

Reply:
It is `optional, when you want to use non-standard implementation` (I added this more explicit description into readme)